asteroid/data/fuss_dataset.py

```diff
@@ -30,21 +30,26 @@ def __init__(self, file_list_path, return_bg=False):
 
         # Load the file list as a dataframe
         # FUSS has a maximum of 3 foregrounds, make column names
-        names = ['mix', 'bg'] + [f'fg{i}' for i in range(self.max_n_fg)]
+        self.fg_names = [f'fg{i}' for i in range(self.max_n_fg)]
```

Comment:
What is your current performance that you get? did you manage to make it work?

Reply:
Like -12dB on validation but it's a stupid value because the net just learns to output zeros where needed (clipped to -20dB). 
Didn't manage yet, don't know why.

Reply:
So you mean the loss is the problem where multiple sources might be inactive?

Reply:
Not sure. I also filtered the data to include only mixtures with all sources active (to check if the inactive loss is the pb) but the loss didn't decrease very much. I have to investigate more and I didn't make much time for it recently.